plugins/image.md

```diff
@@ -280,6 +280,28 @@ tinymce.init({
 });
 ```
 
+### `image_file_types`
+
+This option configures which image file formats will be uploaded by the `image` plugin.
+
+**Type:** `String`
+
+**Default Value:** `'jpeg,jpg,jpe,jfi,jif,jfif,png,gif,bmp,webp'`
```

Comment:
should `tif` be here? same as https://github.com/tinymce/tinymce-docs/pull/1758 ?

Reply:
I added it, it's `jif`